deploy/common/deploy.sh

```diff
@@ -150,7 +155,7 @@ function get_cluster_zip {
       exit 1
     fi
 
-    ID="$(cat "${TMP}" | jq -r .cluster.id)"
+    ID="$(command < "${TMP}" | jq -r .cluster.id)"
```

Comment:
This would fail.

I created sample file "test.json", with following content:
```json
{
        "cluster": {
                "id": 100
        }
}
```
and performed following test:

- `cat test.json | jq -r .cluster.id`
Result: `100`
- command < "${TMP}" | jq -r .cluster.id
No Output
- `jq -r .cluster.id test.json`
Result: `100`

It's better to convert to jq command with filename as argument, instead of cat.

Reply:
@dastgirp Modified all commands with cat and JQ to command JQ with filename as argument